docs/webhooks.rst

```diff
@@ -93,6 +93,34 @@ GitLab
 * Leave the default **Push events** selected and mark **Tag push events** also
 * Finish by clicking **Add Webhook**
 
+Gitea
+~~~~~
+
+.. warning::
+
+   This isn't officially supported, but using the "GitHub webhook" is an effective workaround,
+   because Gitea uses the same payload as GitHub. The generic webhook is not compatibile with Gitea.
+
+* Manually create a "GitHub webhook" integration on Read the Docs
+  (this will show a warning about the webhook not being correctly set up,
+  that will go away when the webhook is configured in Gitea)
+* Go to the :guilabel:`Settings` > :guilabel:`Webhooks` page for your project on your Gitea instance
+* Create a new webhook of type "Gitea" 
```

Comment:
Would it make sense to split this list of steps in two? The ones that are done on RTD and the ones done at Gitea?